Sources/MapboxDirections/DirectionsCredentials.swift

```diff
@@ -1,7 +1,8 @@
 import Foundation
 
 /// The Mapbox access token specified in the main application bundle’s Info.plist.
-let defaultAccessToken = Bundle.main.object(forInfoDictionaryKey: "MGLMapboxAccessToken") as? String
+let defaultAccessToken = Bundle.main.object(forInfoDictionaryKey: "MBXAccessToken") as? String
+let fallbackAccessToken = Bundle.main.object(forInfoDictionaryKey: "MGLMapboxAccessToken") as? String
```

Comment:
Consolidate this into a single variable with a `??` chain, so we can be sure no future code will check `MBXAccessToken` without also checking `MGLMapboxAccessToken`.

Reply:
Done.